frontend/ui/elements/common_info_menu_table.lua

```diff
@@ -51,38 +52,17 @@ common_info.about = {
         })
     end
 }
-common_info.debug_logging = {
-    text = _("Enable verbose debug logging"),
+common_info.report_bug = {
+    text = _("Report a bug"),
     checked_func = function()
         return G_reader_settings:isTrue("debug_verbose")
     end,
-    callback = function()
-        -- Unlike in the dev options, we flip everything at once.
-        if G_reader_settings:isTrue("debug_verbose") then
-            dbg:setVerbose(false)
-            dbg:turnOff()
-            G_reader_settings:makeFalse("debug_verbose")
-            G_reader_settings:makeFalse("debug")
-        else
-            dbg:turnOn()
-            dbg:setVerbose(true)
-            G_reader_settings:makeTrue("debug")
-            G_reader_settings:makeTrue("debug_verbose")
-        end
-        -- Also unlike the dev options, explicitly ask for a restart,
-        -- to make sure framebuffer pulls in a logger.dbg ref that doesn't point to noop on init ;).
-        UIManager:askForRestart()
-    end,
-}
-common_info.report_bug = {
-    text = _("Report a bug"),
-    keep_menu_open = true,
-    callback_func = function()
+    callback = function(touchmenu_instance)
         local DataStorage = require("datastorage")
         local log_path = string.format("%s/%s", DataStorage:getDataDir(), "crash.log")
         local common_msg = T(_("Please report bugs to \nhttps://github.com/koreader/koreader/issues\n\nVersion:\n%1\n\nDetected device:\n%2"),
             Version:getCurrentRevision(), Device:info())
-        local log_msg = T(_("Reproduce the issue with verbose debug logging enabled, and attach %1 to your bug report."), log_path)
+        local log_msg = T(_("Closing this popup will toggle verbose debug logging, reproduce the issue while it's enabled, and attach %1 to your bug report."), log_path)
```

Comment:
Feels a bit authoritarian to force that on a user :) I could just tap tihs menu item to see what's proposed - and I'll get stuck with verbose debug logs, that I'll have to find out how to disable (or not care and fill my device with logs) :/

(And somehow, not all report needs a crash.log - and we'll get 5Mb crash.log that we will have to look at even if you guess from the issue description they won't give any insight, just in case.)

Reply:
> Regardless of that, yeah, I'm not entirely sold on merging those two myself, but I get @Frenzie's original concern, too; and I don't think adding yet another prompt to make it optional would be any better...

I just noticed that I misread Frenzie's poposition though, which was a confirmbox, that I somehow read as a checkmark, for some reason xD.

That probably makes more sense, going to try that ;).

Reply:
Now looks like this: 
![unibutton](https://github.com/koreader/koreader/assets/111974/51e7960e-9a60-420e-b37a-52a4ae5c0c9e)

I went with an unibutton because it looked cramped otherwise, and we still have ample space to tap outside to dismiss.

Reply:
Not sure if it wouldn't be clearer to have a `text_func` to append a spelled out "(logging enabled)" instead of the checkmark?

Reply:
> Not sure if it wouldn't be clearer to have a `text_func` to append a spelled out "(logging enabled)" instead of the checkmark?

Eeh, not a fan.

```diff
diff --git a/frontend/ui/elements/common_info_menu_table.lua b/frontend/ui/elements/common_info_menu_table.lua
index 6d85941fb..b4b2562c8 100644
--- a/frontend/ui/elements/common_info_menu_table.lua
+++ b/frontend/ui/elements/common_info_menu_table.lua
@@ -54,10 +54,19 @@ common_info.about = {
     end
 }
 common_info.report_bug = {
-    text = _("Report a bug"),
+    text_func = function()
+        local label = _("Report a bug")
+        if G_reader_settings:isTrue("debug_verbose") then
+            label = label .. " " .. _("(verbose logging enabled)")
+        end
+        return label
+    end,
+    keep_menu_open = true,
+    --[[
     checked_func = function()
         return G_reader_settings:isTrue("debug_verbose")
     end,
+    --]]
     callback = function(touchmenu_instance)
         local DataStorage = require("datastorage")
         local log_path = string.format("%s/%s", DataStorage:getDataDir(), "crash.log")
```